docs/deploy-using-sepolia.org

```diff
@@ -111,154 +111,161 @@ cast wallet new-mnemonic
-#+end_src
+This process will take longer than the typical startup time mostly
+because the block interval on Sepolia is longer than the default block
+interval for our fake L1. As of Oct 16, 2024, this process takes
+around 9 minutes to finish.
 
-You should expect to see the below logs once the batches have been verified on Sepolia.
-#+begin_src bash
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:11:56.349] [5/13 Execution] Starting sequencing stage 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:11:56.363] [5/13 Execution] Starting batch 361... 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:11:56.363] [5/13 Execution] Starting block 911 (forkid 9)... 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:11:56.363] [5/13 Execution] Waiting for txs from the pool... 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.376] [5/13 Execution] Increment trie hashes started previousRootHeight=910 calculatingRootHeight=911
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.377] [5/13 Execution] Regeneration trie hashes finished. Commiting batch 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.377] [5/13 Execution] Increment ended 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.377] [5/13 Execution] Finish block 911 with 0 transactions... 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.380] [5/13 Execution] Starting block 912 (forkid 9)... 
-[cdk-erigon-sequencer-001] [INFO] [08-29|02:12:02.380] [5/13 Execution] Waiting for txs from the pool... 
-#+end_src
 
-Now the CDK Node container can be restarted
+To make sure that the network is functional, you can run the script
+below to send a test transaction and to ensure that your network is
+progressing as expected.
+
 #+begin_src bash
-docker start <CDK_Node_Container_ID>
-#+end_src
+rpc_url=$(kurtosis port print cdk cdk-erigon-node-001 rpc)
+zkevm_l2_admin_private_key="0x69659a88bd0950e4914f69aaa867e258ff39bc6274d1576a23c477c65987e31f"
 
-You should see the Kurtosis deployment continuing and finishing shortly after.
+cast rpc --rpc-url $rpc_url zkevm_batchNumber
+cast rpc --rpc-url $rpc_url zkevm_virtualBatchNumber
+cast rpc --rpc-url $rpc_url zkevm_verifiedBatchNumber
 
-To make sure that the network is functional, you can query the below commands, and after approximately 20 minutes, you should see the virtualBatchNumber and verifiedBatchNumber incrementing.
-#+begin_src bash
-cast rpc zkevm_batchNumber 
-cast rpc zkevm_virtualBatchNumber 
-cast rpc zkevm_verifiedBatchNumber 
+cast send --legacy --rpc-url $rpc_url --private-key $zkevm_l2_admin_private_key --value 1 0x0000000000000000000000000000000000000000
 #+end_src
 
-Once this completes, you'll probably want to see your deployed
+At this point, you'll probably want to see your deployed
 contracts an on-chain activity.
 
 #+begin_src bash :exports both verbatim :results output code
-kurtosis service exec cdk-v1 contracts-001 'cat /opt/zkevm/combined.json'
+kurtosis service exec cdk8 contracts-001 'cat /opt/zkevm/combined.json'
```

Comment:
I think this is a typo `cdk8` -> `cdk`

Reply:
oops. my actual test enclave was cdk8. I'll rename